### PR TITLE
Style suggestions

### PR DESCRIPTION
**Pull request by carolanncan**
* A few small changes to formatting and wording

### CODE REVIEW

File: introduction-to-metalsmith/index.html [hunk 1]
```diff
@@ -59,7 +59,9 @@ <h2 id="what-is-metalsmith-">What is Metalsmith?</h2>
 <p>Metalsmith, at its core, takes files in a source directory, perform operations on them via plugins, and writes those files into a destination directory. Plugins can do virtually anything: create new files, filter out files, modify files based on some logic, etc (note this means that plugin order <em>does</em> matter; transformations done by one plugin can be seen and used by following plugins, making the process a modular build pipeline). The entire <a href="https://github.com/segmentio/metalsmith.io/blob/master/src/plugins.json">ecosystem of plugins</a> is what makes Metalsmith so flexible.</p>
 <p><em>Note that you have control over telling Metalsmith the location of the source and destination directories.</em></p>
 <h2 id="how-does-metalsmith-and-its-plugins-work-">How does Metalsmith (and its plugins) work?</h2>
-<p>For example, let&#x2019;s see what Metalsmith does if there are no plugins. Here is the directory (<code>./node_modules</code> is hidden):</p>
+<p>For example, let&#x2019;s see what Metalsmith does if there are no plugins.
+Here is the directory
```
Comment (carolanncan) [inline introduction-to-metalsmith/index.html]: Dumb question - should I create those files?

Comment (carolanncan) [inline introduction-to-metalsmith/index.html]: Another dumb question - how do I start the server to view my project locally?

File: introduction-to-metalsmith/index.html [hunk 2]
```diff
@@ -81,8 +83,8 @@ <h2 id="how-does-metalsmith-and-its-plugins-work-">How does Metalsmith (and its
 |   &#x2514;&#x2500;&#x2500; nested
 |       &#x2514;&#x2500;&#x2500; index.md
 &#x2514;&#x2500;&#x2500; index.js
-</code></pre><p>..nothing has changed (not even the text inside the markdown files). This is expected since no plugins (i.e. transforms) are being performed.</p>
-<p>Let&#x2019;s dig one level deeper and see what Metalsmith is seeing to help us build a better picture of what&#x2019;s going on. We&#x2019;ll add this really simple function (hint: it&#x2019;s a plugin!) that just logs some info:</p>
+</code></pre><p>Nothing has changed (not even the text inside the markdown files). This is expected since no plugins (i.e. transforms) are being performed.</p>
+<p>Let&#x2019;s dig one level deeper to build a better picture of what&#x2019;s going on. We&#x2019;ll add this really simple function (hint: it&#x2019;s a plugin!) that just logs some info so we can see what Metalsmith is seeing:</p>
 <pre><code class="lang-javascript">var Metalsmith = require(&apos;metalsmith&apos;);
 
 Metalsmith(__dirname)
```
Comment (carolanncan) [inline introduction-to-metalsmith/index.html]: Would be nice to have a commented line that indicates:

`//Your second plug in starts here`

**Comment by lambtron** [comment]
Ah, you are starting to peel back the onion of the mysterious `plato`.

First, you would need to [install `plato`](https://github.com/segmentio/plato/blob/master/docs/install.md).

Then, clone this repo. In that directory, run `plato preview`.

Let me know if that works :)

selfie-0

**Pull request closed without merging**